doc/api/cli.md

```diff
@@ -3306,6 +3306,12 @@ If `value` equals `'child'`, test reporter options will be overridden and test
 output will be sent to stdout in the TAP format. If any other value is provided,
 Node.js makes no guarantees about the reporter format used or its stability.
 
+### `NODE_TEST_WORKER_ID`
```

Comment:
I'm not sure this is the correct place for this. this is a list of env vars node accepts, not env vars it exposes